reference/7.0/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -496,6 +496,108 @@ in simple conditional cases.
 
 For more information, see [about_If](about_If.md).
 
+#### Null-coalescing operator `??`
+
+The null-coalescing operator `??` returns the value of its left-hand operand if it isn't null;
```

Comment:
Change the semicolon to period and start a new sentence.

Reply:
Fixed